### ISSUE TITLE
swagger/wsdl url or file to import backend api directly in contract json file

### ISSUE THREAD

**Issue by gcornacchia** [Axway-API-Management-Plus/apimanager-swagger-promote#38]
I have a suggestion, what do you think about writing the swagger file/url or wsdl url (for importing the backend api) directly in the contract json file?

The same way is configured for certificates with the relative or absolute path of certificate to trust for a specific API.

Something like that:
{
	"name": "CalculatorService_v1",
	"path": "/CalculatorService/v1",
        **"url":"http://www.dneonline.com/calculator.asmx",**
	"state":"published",
	"version": "1.0.0",
	"organization": "XXX" 
}

or 

{
	"name": "Petstore",
	"path": "/petstore/v1",
        **"import":"./opt/temp/petstore.json",**
	"state":"published",
	"version": "1.0.0",
	"organization": "XXX" 
}

In this way I can version my contract file (together with swagger file for example) which is the only relevant input for the tool swagger-promote.

We could keep both the solutions, if no '-a' parameter is passed, read from the contract json file, if present.

If you want, I can work on it by submitting a pull request.

**Comment by cwiechmann** [comment]
That’s a very good idea, as it will simplify the configuration of the CI/CD workflow again.

I would like to stick to the same name and format, as it is used with -a parameter. Including username/password for URLs.

Like this:
```json
{
"name": "Petstore",
"path": "/petstore/v1",
"apiDefinition":"./opt/temp/petstore.json",
"state":"published",
"version": "1.0.0",
"organization": "XXX"
}
```

**Comment by cwiechmann** [comment]
Released with version 1.5.0.
